src/tools/scripts/creationCoordinateur/resetCoordinateur.js

```diff
@@ -4,41 +4,73 @@
 require('dotenv').config();
 
 const { execute } = require('../../utils');
+const { program } = require('commander');
 
 execute(__filename, async ({ db, logger, exit }) => {
-  logger.info('Reset/Purge de tout les coordos....');
-  // estCoordinateur
-  await db.collection('conseillers').updateMany(
-    { 'estCoordinateur': true },
-    { $unset: { 'estCoordinateur': '' } },
-  );
-  await db.collection('misesEnRelation').updateMany(
-    { 'conseillerObj.estCoordinateur': true },
-    { $unset: { 'conseillerObj.estCoordinateur': '' } },
-  );
-  // listeSubordonnes
-  await db.collection('conseillers').updateMany(
-    { 'listeSubordonnes': { '$exists': true } },
-    { $unset: { 'listeSubordonnes': '' } }
-  );
-  await db.collection('misesEnRelation').updateMany(
-    { 'conseillerObj.listeSubordonnes': { '$exists': true } },
-    { $unset: { 'conseillerObj.listeSubordonnes': '' } }
-  );
-  // tag coordinateurs
-  await db.collection('conseillers').updateMany(
-    { 'coordinateurs': { '$exists': true } },
-    { $unset: { 'coordinateurs': '' } }
-  );
-  await db.collection('misesEnRelation').updateMany(
-    { 'conseillerObj.coordinateurs': { '$exists': true } },
-    { $unset: { 'conseillerObj.coordinateurs': '' } }
-  );
-  // Role
-  await db.collection('users').updateMany(
-    { 'roles': { '$in': ['coordinateur_coop'] } },
-    { $pull: { 'roles': 'coordinateur_coop' } }
-  );
-  logger.info('Fin de la purge.');
+  program.option('-i, --id <id>', 'id: idPG du conseiller');
+  program.option('-r, --reset', 'reset du coordo');
+  program.helpOption('-e', 'HELP command');
+  program.parse(process.argv);
+
+  const { id, reset } = program;
+  const conseiller = await db.collection('conseillers').findOne({ idPG: ~~id });
+  if (!conseiller) {
+    logger.error(`Le conseiller ${~~id} n'existe pas`);
+    return;
+  }
+  const user = await db.collection('users').findOne({ 'entity.$id': conseiller._id });
```

Comment:
yes on peut même ajouter roles: {$in: ['coordinateur_coop] } A voir dans le TdB on va le renommer en coordinateur tout court d'ailleurs ou alors on met les 2 pour anticiper

Reply:
https://github.com/anct-cnum/api-conseiller-numerique/pull/1318/files#r1433858115